Documentation/InputSimulation/InputSimulationService.md

```diff
@@ -0,0 +1,153 @@
+# Input Simulation Service
+
+The Input Simulation Service emulates the behaviour of devices and platforms that may not be available in the Unity editor. Examples include:
+* Hololens or VR device head tracking
+* Hololens version 1 hand gestures
+* Hololens version 2 articulated hand tracking
+
+Users can use a conventional keyboard and mouse combination to control simulated devices at runtime. This allows testing of interactions in the Unity editor without first deploying to a device.
+
+## Enabling the Input Simulation Service
+
+Input simulation is enabled by default in MRTK.
+
+Input simulation is an optional [Mixed Reality service](../../External/Documentation/MixedRealityServices.md). It can be added or removed in to the [Registered Service Providers profile](../TODO.md). Select _Microsoft.MixedReality.Toolkit.Input > InputSimulationService_ as the __Component Type__. The __Runtime Platform__ should always be _Windows Editor_ since the service depends on keyboard and mouse input. The Input Simulation Service has its own profile linked under __Configuration Profile__.
+
+| __Warning__: Any type of profile can be assigned to services at the time of this writing. If you assign a different profile to the service, make sure to use a profile of type _Input Simulation_ or it will not work! |
+| --- |
+
+![Default Input Simulation service settings](../Images/MRTK_InputSimulation_RegisteredServicesEnable.png)
```

Comment:
Can you confirm that this is still the case after Dave submits his changes here:

https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3738/files#diff-72713bda47fb5e09eb55e5066ef13b29

i.e. I see a huge swath of configurations getting axed, one of them being Input Simulation

Reply:
@davidkline-ms

Reply:
That will indeed need to change. The input simulation service will be registered in the input system profile going forward (and in fact is there by default)

Reply:
The InputSim service was moved to the Input System profile, but

1. The profile for Input Simulation Service is still not type-specific, so the warning is still valid IMO. You can select any of the 20-30 existing default profiles to link here. The service has an internal check, so it will simply not activate, but the asset selection is not filtering profiles by type.
I will have to update the images and paths for accessing the profile though.

2. I don't see the service (aka. data provider - what's the difference again?) being registered by default. I had to add one manually to get hand sim back.